src/lib.rs

```diff
@@ -129,6 +139,10 @@ pub struct Args {
     /// relative path and only try a GitHub URL if the local path doesn't exist.
     #[structopt(short, long)]
     pub git: Option<String>,
+    /// Folder to copy the template from, or in case the --git option has been specified,
+    /// the sub-folder within the git repository from which to copy the template.
+    #[structopt(long)]
+    pub folder: Option<PathBuf>,
```

Comment:
could you please rename `folder` to `dir`? 
IMO This small change would improve the typing a bit, since `--dir` is as along as `--git`

Reply:
I did find 1 issue that might have been the cause. At least it is the only one I could find: I forgot to exclude any `.git` folder when copying over files.

That has now been fixed.

Also renamed `folder` to `dir` as requested.

I actually found it confusing when people started calling directories for folders (wonder where that came up, because AFAIK it used to be only directory or file).
Now I don't know when to call it a folder or a directory - as Walter (Jeff Dunham) would say: pisses me off!